src/main/kotlin/no/nav/syfo/application/ApplicationEnvironment.kt

```diff
@@ -15,6 +15,16 @@ data class Environment(
     val isoppfolgingstilfelleDbName: String = getEnvVar("${NAIS_DATABASE_ENV_PREFIX}_DATABASE"),
     val isoppfolgingstilfelleDbUsername: String = getEnvVar("${NAIS_DATABASE_ENV_PREFIX}_USERNAME"),
     val isoppfolgingstilfelleDbPassword: String = getEnvVar("${NAIS_DATABASE_ENV_PREFIX}_PASSWORD"),
+
+    val kafka: ApplicationEnvironmentKafka = ApplicationEnvironmentKafka(
+        aivenBootstrapServers = getEnvVar("KAFKA_BROKERS"),
```

Comment:
Må legge til litt konfig i nais-filene for dette (men det kommer kanskje senere?):
  kafka:
    pool: nav-dev

Reply:
Stemmer, det glemte jeg å legge til her. Ordner nå!

Reply:
Fikset.